src/en/readonepunchmanmangaonlinetwo/src/eu/kanade/tachiyomi/extension/en/readonepunchmanmangaonlinetwo/ReadOnePunchManMangaOnlineTwo.kt

```diff
@@ -25,6 +30,12 @@ class ReadOnePunchManMangaOnlineTwo : MangaCatalog("Read One-Punch Man Manga Onl
     override fun chapterFromElement(element: Element): SChapter = SChapter.create().apply {
         name = element.select("td:first-child").text()
         url = element.select("a").attr("abs:href")
-        date_upload = System.currentTimeMillis() // I have no idear how to parse Date stuff
+        date_upload = parseDate(element.select("td:nth-child(2)").text())
+    }
+
+    private fun parseDate(date: String) = DATE_FORMAT.parse(date)?.time ?: System.currentTimeMillis()
```

Comment:
Use the `tryParse()` function. Don't invent your own `parseDate()`.

https://github.com/keiyoushi/extensions-source/blob/98a4e1540993d487aa254e7626bb688d06731cda/core/src/main/kotlin/keiyoushi/utils/Date.kt#L7